src/tests/modules/python/auth_with_shared_storage.unlang

```diff
@@ -1,13 +1,13 @@
 # Should return noop (no shared variable set)
-pmod_shared_storage
+pmod7_shared_storage
```

Comment:
Why this change?  I don't understand...

Reply:
Just to keep the same as the previous tests with numbers. e.g: `pmod${number}_$name`